react-components/src/components/RevealCanvas/RevealCanvas.tsx

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { type ReactNode, type ReactElement, useRef, useEffect } from 'react';
+import { createPortal } from 'react-dom';
+import { useReveal } from './ViewerContext';
+
+export function RevealCanvas({ children }: { children?: ReactNode }): ReactElement {
+  const viewer = useReveal();
+  const parentElement = useRef<HTMLDivElement>(null);
+
+  return (
```

Comment:
I feel below look more clean, what do you think?
`useEffect(() => {`
`    if (parentElement.current !== null) {`
`      parentElement.current.appendChild(viewer.domElement);`
`    }`
`  }, []);`

`  return (`
`    <div style={{ width: '100%', height: '100%', overflow: 'hidden' }} ref={parentElement}>`
`      {createPortal(children, viewer.domElement)}`
`    </div>`
`  );`

Reply:
I agree :+1: Will change